tests/vmi_multus_test.go

```diff
@@ -1335,12 +1379,16 @@ func getInterfaceNetworkNameByMAC(vmi *v1.VirtualMachineInstance, macAddress str
 // Set creates a linux bridge and configures the firewall. We use iptables-compat in order to work with
 // both iptables and newer nftables.
 // TODO: Once kubernetes-nmstate is ready, we should use it instead
-func configureNodeNetwork(virtClient kubecli.KubevirtClient) {
+func configureNodeNetwork(virtClient kubecli.KubevirtClient) (*func() (*appsv1.DaemonSet, error), error) {
 
 	// Fetching the kubevirt-operator image from the pod makes this independent from the installation method / image used
 	pods, err := virtClient.CoreV1().Pods(flags.KubeVirtInstallNamespace).List(context.Background(), metav1.ListOptions{LabelSelector: "kubevirt.io=virt-handler"})
-	Expect(err).ToNot(HaveOccurred())
-	Expect(pods.Items).ToNot(BeEmpty())
+	if err != nil {
+		return nil, err
+	}
+	if len(pods.Items) == 0 {
+		return nil, fmt.Errorf("Pods list should not be empty")
```

Comment:
The error here is: "no virt-handler pods detected".

Note that with errors, it is common not to start with capital letters, as these many times chained to other error text.

Reply:
Yes, overlooked that. Thanks